src/extractor/guidance/coordinate_extractor.cpp

```diff
@@ -352,40 +370,25 @@ util::Coordinate CoordinateExtractor::ExtractRepresentativeCoordinate(
          * We distinguish between turns that simply model the initial way of getting onto the
          * destination lanes and the ones that performa a larger turn.
          */
-        const double offset =
-            std::min(0.5 * considered_lanes * ASSUMED_LANE_WIDTH, 0.2 * segment_distances.back());
-        coordinates = TrimCoordinatesToLength(std::move(coordinates), offset, segment_distances);
+        coordinates =
+            TrimCoordinatesToLength(std::move(coordinates),
+                                    2 * distance_to_skip_over_due_to_coordinate_inaccuracies,
```

Comment:
I dunno, I actually think this is helpful naming - in 6 months from now, when someone else is looking at this code, the intent is pretty clear.

Makes it easy to search for too :-)